lib/zuul.js

```diff
@@ -78,9 +74,17 @@ Zuul.prototype.run = function (cb) {
 
   var config = self._config
 
-  self._setup(function (err, control_port) {
+  self._setup(function (err, server) {
```

Comment:
First attempt was to pass the server object so we could close it. It turns out that `control-app.js` also contains an instance of `watchify`. See future commit.